easybuild/easyconfigs/c/csvkit/csvkit-1.0.4-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,82 @@
+easyblock = 'PythonBundle'
+
+name = 'csvkit'
+version = '1.0.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/wireservice/csvkit'
+description = """csvkit is a suite of command-line tools for converting to and working with CSV,
+ the king of tabular file formats."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [
+    ('Python', '3.7.2'),
```

Comment:
I got to ask: multi_deps? The software seems to support python2.7 at least.

Reply:
@Micket `multi_deps` does not make sense here, because the "binaries" will be made for only 1 version of python.
what's worse: in the second iteration the binaries are overwritten with the second python version, but when the module is loaded the first python is loaded by default, so the binaries look for the wrong libraries and fail.

Reply:
Don't we have something that replaces the hash bang in python scripts? The code in the scripts seems like they would be python2 and 3 compatible.